fabtests/Makefile.am

```diff
@@ -391,7 +391,7 @@ dist-hook: fabtests.spec
 	cp fabtests.spec $(distdir)
 
 test:
-	./scripts/runfabtests.sh -vvv -S $(os_excludes)
+	./scripts/runfabtests.sh -vvv -S $(os_excludes) -R -f ./test_configs/sockets/sockets.exclude sockets
```

Comment:
I wonder if there's a way to get the exclude file from the location where it would be installed (e.g. /usr/share/fabtests/test_configs/)

Reply:
Maybe?  This part just copied the same syntax used by the other lines.  But I can update the script to also search /usr/share and /usr/local/share for exclude files.

Reply:
There's one potential problem though. If the installed config (in /usr/local) is from an older version of libfabric, the user might be unaware of that.

A better way of doing this would be to get the installation location from configure and use that. Not sure how easy is that. I think the present code that just searches the current working directory is good enough.